.github/workflows/track-contributor-prs.yml

```
@@ -0,0 +1,23 @@
+  pull-requests: read
+
+jobs:
+  update-pr-count:
+    if: github.event.pull_request.merged == true
+    uses: StabilityNexus/ContributorAutomation/.github/workflows/reusable-update-pr.yml@main
+    with:
+      pr_number: ${{ github.event.pull_request.number }}
+      repo_name: ${{ github.repository }}
+      pr_author: ${{ github.event.pull_request.user.login }}
+      pr_title: ${{ github.event.pull_request.title }}
+      lines_changed: ${{ github.event.pull_request.additions + github.event.pull_request.deletions }}
```

actionlint (1.7.10)

got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)